app/importers/open_access_button_publication_importer.rb

```diff
@@ -70,4 +73,8 @@ def query_open_access_button_for(publication)
         }
       )
     end
+
+    def doi
+      DOISanitizer.new(doi_element).url || DOISanitizer.new(url).url || DOISanitizer.new(issn).url
+    end
```

Comment:
I think this can be removed since it isn't doing anything.

Reply:
Yep, my bad, I meant to delete that